client/js/models/OurOfferings.js

```diff
@@ -0,0 +1,17 @@
+module.exports = { ...require('./__proto__.js'),
+	data: [
+		{header: 'fall 2017', item: '', quantity: '', price: ''},
+		{header: '', item: 'Carrots', quantity: 'Bushel', price: '$ 1.50'},
```

Comment:
I'd avoid all these empty header properties. Actually, I'd get rid of the property altogether. You have a clear justification for having two data sets here:  one is for fall, one is for year round. So organize the data like this:
`data: { fall: [ ...fall items ], year-round: [ ...year-round items ] }`